kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/config/ReThinkSinkSettings.scala

```diff
@@ -69,7 +69,18 @@ object ReThinkSinkSettings {
     val ignoreFields = config.getIgnoreFields(routes)
     val retry = config.getInt(ReThinkSinkConfigConstants.ERROR_RETRY_INTERVAL).toLong
 
-    ReThinkSinkSetting(db, routes, topicTableMap, fieldMap, ignoreFields, p, conflictMap, errorPolicy, maxRetries, retry, batchSize)
+    ReThinkSinkSetting(
+      db,
+      routes,
+      topicTableMap,
+      fieldMap,
+      ignoreFields,
+      p,
```

Comment:
Yeah, I just went past that change and was coming here to comment back =)

Cheers